shipyard_blueprints/microsoft-power-bi/shipyard_microsoft_power_bi/cli/trigger_sync.py

```diff
@@ -0,0 +1,54 @@
+import sys
+import argparse
+from shipyard_microsoft_power_bi import MicrosoftPowerBiClient
+from shipyard_templates.exit_code_exception import ExitCodeException
+
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--client-id", dest="client_id", required=True)
+    parser.add_argument("--client-secret", dest="client_secret", required=True)
+    parser.add_argument("--tenant-id", dest="tenant_id", required=True)
+    parser.add_argument("--group-id", dest="group_id", required=True)
+    parser.add_argument("--object-type", dest="object_type", required=True)
+    parser.add_argument("--object-id", dest="object_id", required=True)
+    parser.add_argument(
+        "--wait-for-completion", dest="wait_for_completion", required=True
+    )
+    parser.add_argument("--poke-interval", dest="poke_interval", required=False)
```

Comment:
Set a default value of this for 60 seconds, since that is what we will do in the application